### DISCUSSION TITLE
Coloring raster tiles

### DISCUSSION THREAD

**Discussion by Maneesh7012** [protomaps/PMTiles#316] [Q&A]
Hi, Is it possible to color raster tiles based on pixel value?

**Comment by bdon** [comment]
You can store raster tiles, RGB or other raw band data, in a PMTiles archive and display them in a renderer like Leaflet, OpenLayers or MapLibre GL, DeckGL, etc. Those renderers may have coloring features for raster data but that's outside of the scope of PMTiles which is the tile container format and storage + serving ecosystem.
